### DISCUSSION TITLE
Azure Batch ‘CloudServiceConfiguration’ pools will be retired on 29 February 2024

### DISCUSSION THREAD

**Discussion by gfchaves** [azure-deprecation/dashboard#126] [Deprecation]
Azure Batch ‘CloudServiceConfiguration’ pools will be retired on 29 February 2024

https://azure.microsoft.com/en-us/updates/azure-batch-cloudserviceconfiguration-pools-will-be-retired-on-29-february-2024/

**Comment by tomkerkhove** [comment]
Yes, it's on my radar along with 6 other ones 😅

**Reply by tomkerkhove** [reply]
![image](https://user-images.githubusercontent.com/4345663/108845512-cf3c7480-75dd-11eb-8ecc-05b3753fe58d.png)

**Reply by gfchaves** [reply]
ok, and about the AKS legacy Azure AD integration will be retired on 29 February 2024 =>  https://azure.microsoft.com/en-us/updates/aks-legacy-azure-ad-integration-will-be-retired-on-29-february-2024/

**Reply by tomkerkhove** [reply]
- https://github.com/azure-deprecation/dashboard/issues/127
- https://github.com/azure-deprecation/dashboard/issues/128

The flood is coming tomorrow, thanks for helping!
